grants/milestone_deliveries/season_2/package_manager/milestone1-part1-details.md

```diff
@@ -0,0 +1,189 @@
+# Radicle Package Releases Research Notes
+
+## Overview
+This gist is a summary and blog of research notes I took while researching the possibility of using Radicle as a platform for generating and distributing software packages, such as npm packages. This is part of a [research grant through Radicle](https://radicle.community/t/grant-application-package-manager/3122).
+
+Here is a short summary video of this gist:
+
+[![Watch the video](https://img.youtube.com/vi/RmfZo2QdV0A/hqdefault.jpg)](https://youtu.be/RmfZo2QdV0A)
```

Comment:
This is a really, really awesome video. 

Super clear and succinct summary of all the points :D